build-system/release-tagger/test/update-release.publish.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+process.argv.push('--action', 'publish', '--tag', '2007210123000');
```

Comment:
Curious: Will these flags accumulate when multiple tests are run in the same `amp ava` invocation? If so, would it work if the arguments were set in `test.before()` and cleaned up in `test.after()`?

Reply:
good call. decided to not push to argv at all. thanks!

Reply:
(it wouldn't work to be set in `test.before()` because it needs to be set before loading `update-release.js`. And loading that file in `test.before()` doesn't work for nock